packages/frontend-core/src/components/grid/cells/RelationshipCell.svelte

```diff
@@ -322,6 +357,21 @@
   </GridPopover>
 {/if}
 
+{#if relationshipFields}
+  <GridPopover open={showRelationshipFields} {anchor}>
```

Comment:
NAB: I think we should increase with min width here so that average length text values have more room 👍  Something like `minWidth={300}`, or more, depending what you think looks best.

Reply:
Done: 
<img width="472" alt="image" src="https://github.com/user-attachments/assets/8674e55f-41f9-42ac-b3f7-863d0ad3cbda">